Project.toml

```
@@ -21,7 +21,7 @@ PDMats = "0.9"
 QuadGK = "2"
 SpecialFunctions = "0.8, 0.9"
 StatsBase = "0.32"
-StatsFuns = "0.8, 0.9"
+StatsFuns = ">= 0.9.3"
```

this is required by the use of a function in StatsFuns

---

Then the new convention is this:
```suggestion
StatsFuns = "0.9.3"
```

CompatHelper is there to bump the dependency when a new version is tagged.